src/Core/Settings.h

```diff
@@ -124,6 +124,10 @@ class IColumn;
     M(UInt64, parallel_replicas_count, 0, "", 0) \
     M(UInt64, parallel_replica_offset, 0, "", 0) \
     \
+    M(Bool, enable_sample_offset_parallel_processing, false, "If true, ClickHouse will send a SELECT query to all replicas of a table. It will work only for tables with SAMPLE key.", 0)\
+    M(Bool, parallel_reading_from_replicas, false, "If true, ClickHouse will send a SELECT query to all replicas of a table. It will work for any kind on MergeTree table.", 0) \
```

Comment:
I think, we need only one setting `parallel_reading_from_replicas`.
`enable_sample_offset_parallel_processing = !parallel_reading_from_replicas && max_parallel_replicas > 1`
`collaborate_with_initiator = parallel_reading_from_replicas && max_parallel_replicas > 1`